backend/apps/ai/management/commands/ai_create_chapter_chunks.py

```
@@ -0,0 +1,211 @@
+        else:
+            queryset = Chapter.objects.filter(is_active=True)
+
+        if not (total_chapters := queryset.count()):
+            self.stdout.write("No chapters found to process")
+            return
+
+        self.stdout.write(f"Found {total_chapters} chapters to process")
+
+        batch_size = options["batch_size"]
+        for offset in range(0, total_chapters, batch_size):
+            batch_chapters = queryset[offset : offset + batch_size]
```

Flake8 (7.2.0)

[error] whitespace before ':'

(E203)